src/docs/content/observability/observability-scenario/pre-requisites/oci-logging.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "OCI Logging"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 3
+tags:
+  - oci logging
+---
+
+### Introduction
+
+The Oracle Cloud Infrastructure (OCI) Logging service is a highly scalable and fully managed single pane of glass for all the logs in a tenancy. 
+
+In this section we will be enabling OCI (Oracle Cloud Infrastructure) logging for OKE (Oracle Cloud Infrastructure Container Engine for Kubernetes) worker nodes. 
+
+All the OKE pod logs (stdout and stderr) are written to /var/log/containers in a JSON format by default.
+
+Example:  
+```
+Log File Name Format: <Pod>_<NameSpace>_<ContainerName>-<ContainerID>.log
+Log File Content Format:  {"log":"<message>","stream":"<stdout|stderr>","time":"<timestamp>"}
+```
+
+{{% alert style="warning" icon="warning" %}}
+Ensure that the worker nodes are running OL 7.6 and above.
+{{% /alert %}}
+
+### Pre-Requisites
```

Comment:
now we have prerequisites ->  prerequisites... This content would be better shown as steps (not unlike the cloud deployment in these docs)

Reply:
Alternatively you can have a TOC like this:

- Observability
  - Introduction
  - Setup
  - Use case or example (instead of scenario)
  - Summary or conclusion

Reply:
Please validate if the new structure seems appropriate.